content/en/docs/external-add-ons/elyra/github.md

```diff
@@ -0,0 +1,9 @@
++++
+title = "GitHub Repository"
+description = "GitHub repository for Elyra"
+weight = 999
+manualLink = "https://github.com/elyra-ai/elyra"
+icon = "fa-brands fa-github"
++++
+
+Elyra is developed in the [`elyra-ai/elyra`](https://github.com/kserve/kserve) repository.
```

Comment:
Incorrect link here.

Reply:
Fixed link in https://github.com/kubeflow/website/pull/3640/commits/5be9ffa805427c5ad1645c5ad879fb1b6c6ab934****